### PR TITLE
Dev mar2016

### PR DESCRIPTION
**Pull request by gschmidt-usgs**
Ready for peer review

### CODE REVIEW

File: ledaps/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-## Ledaps Version 2.4.0 Release Notes
-Release Date: November 23, 2015
+## Ledaps Version 2.5.0 Release Notes
+Release Date: March 23, 2016
```
Comment (ronaldddilley) [inline ledaps/README.md]: Same comment as on spectral indices

File: ledaps/README.md [hunk 2]
```diff
@@ -92,8 +92,11 @@ After compiling the product-formatter raw\_binary libraries and tools, the conve
 ### Product Guide
 
 ## Changes From Previous Version
-#### Updates on November 23, 2015 - USGS EROS
+#### Updates on March 23, 2016 - USGS EROS
```
Comment (ronaldddilley) [inline ledaps/README.md]: Same date comment

File: ledaps/ledapsSrc/src/lndcal/output.c [hunk 3]
```diff
@@ -179,7 +179,7 @@ revision 2.0.1 3/25/2014
 
   /* Determine the scene name */
   strcpy (scene_name, in_meta->band[rep_indx].file_name);
-  mychar = strchr (scene_name, '_');
+  mychar = strrchr (scene_name, '_');
```
Comment (ronaldddilley) [inline ledaps/ledapsSrc/src/lndcal/output.c]: Per comment in spectral indices.... If we add the scene id/name to the XML then you could change this to use that.

**Comment by ronaldddilley** [comment]
Readme files.  Double check that the environment variables are reflected correctly for the new Makefiles.

File: ledaps/ledapsSrc/src/lndsr/output.c [hunk 4]
```diff
@@ -178,9 +178,15 @@ revision on 1/15/2015 by Gail Schmidt, USGS/EROS
 
   /* Determine the scene name */
   strcpy (scene_name, in_meta->band[rep_indx].file_name);
-  mychar = strchr (scene_name, '_');
-  if (mychar != NULL)
-    *mychar = '\0';
+  mychar = scene_name;
```
Comment (ronaldddilley) [inline ledaps/ledapsSrc/src/lndsr/output.c]: Same comment as spectral indices

File: ledaps/ledapsSrc/src/lndsrbm/LS_geoloc_driver.c [hunk 5]
```diff
@@ -6,101 +6,137 @@
...
-  FILE *error_ptr=NULL;
+    char projection[256];
+    float coordinates[8];
+    double parm[13];
+    double radius, lat, lon, dl, ds;
+    double corner[2];
+    int ret;
+    int zonecode, sphercode, rows, cols;
+    float orientationangle, pixelsize, upperleftx, upperlefty;
+    double arg2, arg3;    /* sample/line or lat/long values */
```
Comment (ronaldddilley) [inline ledaps/ledapsSrc/src/lndsrbm/LS_geoloc_driver.c]: Didn't like arg2, and arg3 names at first, but ok.
Your comment is off though it should be "sample/line or long/lat values".

File: not-validated-prototype-l8_sr/README.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
-## L8SR Version 0.4.0 Release Notes
-Release Date: November 23, 2015
+## L8SR Version 0.5.0 Release Notes
+Release Date: March 23, 2016
```
Comment (ronaldddilley) [inline not-validated-prototype-l8_sr/README.md]: Just flagging this if you decide to change the dates.

File: ledaps/ledapsSrc/src/lndsrbm/LS_geoloc.c [hunk 7]
```diff
@@ -17,480 +16,77 @@
  * Modified on 3/12/2013, Gail Schmidt, USGS EROS
  * Updated to add support for the Polar Stereographic projection, in addition
  *   to the existing support for UTM.
+ *
+ * Modified on 1/5/2016, Gail Schmidt, USGS EROS
+ * Given that most of these variables and the code comes from GCTP, we might
+ *   as well just use GCTP vs. copying the code from GCTP to this location.
+ *   That minimizes the maintenance of code.
```
Comment (ronaldddilley) [inline ledaps/ledapsSrc/src/lndsrbm/LS_geoloc.c]: I like this.  Huge code improvement using the library and the new formatting.

**Comment by ronaldddilley** [comment]
No further comments

File: ledaps/README.md (hunk 1 above)
Comment (gschmidt-usgs) [inline ledaps/README.md]: Fixed.

File: ledaps/README.md (hunk 2 above)
Comment (gschmidt-usgs) [inline ledaps/README.md]: Fixed

File: ledaps/ledapsSrc/src/lndcal/output.c (hunk 3 above)
Comment (gschmidt-usgs) [inline ledaps/ledapsSrc/src/lndcal/output.c]: Pulled scene ID from the XML file.

File: ledaps/ledapsSrc/src/lndsr/output.c (hunk 4 above)
Comment (gschmidt-usgs) [inline ledaps/ledapsSrc/src/lndsr/output.c]: Pulled scene ID from the XML file.

File: ledaps/ledapsSrc/src/lndsrbm/LS_geoloc_driver.c (hunk 5 above)
Comment (gschmidt-usgs) [inline ledaps/ledapsSrc/src/lndsrbm/LS_geoloc_driver.c]: Agree - arg2/arg3 isn't descriptive.  Changed this to arg_x and arg_y.  Updated comments.

File: not-validated-prototype-l8_sr/README.md (hunk 6 above)
Comment (gschmidt-usgs) [inline not-validated-prototype-l8_sr/README.md]: Fixed. Removed this reference to the date and called the section Release Notes.

**Comment by gschmidt-usgs** [comment]
Pulled the history from the function headers.  It will be stored via commit comments in Git.

**Pull request merged**